### PR TITLE
Checkbox Added to the Issues Template

### PR DESCRIPTION
**Pull request by parasss19**
## Description 📝

Added the checkboxes in issue template

### CODE REVIEW

**Comment by PiyushKalyanpy** [comment]
Why you have removed the PR template ?? Why there is no issue number in the description ?? and why are you not following the contribution guidelines ??

File: .github/ISSUE_TEMPLATE/bug.yml
Comment (PiyushKalyanpy) [inline .github/ISSUE_TEMPLATE/bug.yml:1]: This is not the correct way of adding the checkboxes. for all the files. I have provided the code here, please update the code for other template also.
```
- type: checkboxes
    id: terms
    attributes:
      label: 'Record'
      options:
        - label: I agree to follow this project's Code of Conduct
          required: true

        - label: I'm a GSSoC'23 contributor

        - label: I want to work on this issue

```

File: tailwind.config.js
Comment (PiyushKalyanpy) [inline tailwind.config.js:1]: Please remove this file from PR.

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml
Comment (parasss19) [reply .github/ISSUE_TEMPLATE/bug.yml:1]: > This is not the correct way of adding the checkboxes. for all the files. I have provided the code here, please update the code for other template also.
> 
> ```
> - type: checkboxes
>     id: terms
>     attributes:
>       label: 'Record'
>       options:
>         - label: I agree to follow this project's Code of Conduct
>           required: true
> 
>         - label: I'm a GSSoC'23 contributor
> 
>         - label: I want to work on this issue
> ```

Hi @PiyushKalyanpy i will make all the required changes I'm outside right now. Thanks for the suggestions

**Comment by PiyushKalyanpy** [comment]
Please share the screenshot (preview) of the updated template. 
You can test it on your own GitHub repository.

**Comment by parasss19** [comment]
> Please share the screenshot (preview) of the updated template. You can test it on your own GitHub repository.

HI @PiyushKalyanpy I added the checkboxes in issues template and due to some YAML error i have to make some changes due to which there are so many commits but now its added properly.

**Comment by PiyushKalyanpy** [comment]
Please add an appropriate title to the pull request.

**Comment by parasss19** [comment]
> Please share the screenshot (preview) of the updated template. You can test it on your own GitHub repository.

Change the title of PR 
Resolved the issue #285

File: tailwind.config.js
Comment (PiyushKalyanpy) [reply tailwind.config.js:1]: please remove this file also

**Comment by PiyushKalyanpy** [comment]
please create a new pr.

**Comment by parasss19** [comment]
> please create a new pr.

Oki I will create new PR with new branch 👍

**Pull request closed without merging**